.github/workflows/firebase-hosting-merge.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: cd components && yarn && cd ../docs && yarn && cd .. && yarn link:components && cd docs && yarn build
-      - uses: FirebaseExtended/action-hosting-deploy@v0
+      - uses: emclaug2/action-hosting-deploy@main
```

Comment:
by the way @emclaug2 is firebase settled now? Or would you recommend us to keep using your fork?

Reply:
I'm like 90% sure the firebase issues are resolved now; if @ektasawant or @bkarambe wants to test it out by reverting this line to check that'd be awesome.

Reply:
Revert the changes working fine 👍